packages/yourloops/models/shoreline.ts

```diff
@@ -48,6 +48,7 @@ enum UserRoles {
   hcp = "hcp",
   caregiver = "caregiver",
   patient = "patient",
+  empty = "",
```

Comment:
I would not have changed this definition just for the signup form.
This "empty" should be limited to signup form only.

Maybe try to change `SignupUser` (packages/yourloops/lib/auth/models.ts) to:
```
export interface SignupUser {
[...]
  accountRole?: UserRoles;
[...]
}
```
and remove from `initialState`: accountRole: UserRoles.caregiver

Reply:
you're right, it was easier for me but maybe not a good solution ... I'm refactoring :)